REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Nettle 0.2
 JSON 0.5
-ZMQ 0.3.1
+ZMQ 0.3.4
```

Comment:
It works, it just has a lot of warnings. I suspect versions earlier than 0.3.1 would work too without any errors, wouldn't they?